test/test_array.py

```diff
@@ -1847,12 +1847,29 @@ def test_branch_operations_on_pure_scalars():
     cl_array.maximum,
     cl_array.minimum,
 ])
-def test_branch_operations_on_nans(ctx_factory, op):
+@pytest.mark.parametrize("special_a", [
+    np.nan,
+    np.inf,
```

Comment:
Does it make sense to add `-np.inf` here?

Reply:
e8c0d023fa0a592b68de52dc709f2f48ed9e4129